### PR TITLE
Add settings

### LINKED ISSUE #10: Visualization service configuration
Implement visualization service configuration through settings (as currently is for `smart-contract-verifier`)

### PR DESCRIPTION
**Pull request by sevenzing**
closes #10

### CODE REVIEW

File: visualizer-server/build.rs [hunk 1]
```diff
@@ -16,6 +16,8 @@ fn compile(
     config
         .service_generator(generator)
         .compile_well_known_types()
+        // Rust cosider some comments as doc-comments and include them in tests
+        .disable_comments(["."])
```
Comment (leviska) [inline visualizer-server/build.rs]: may be we could ignore generated files in doc tests instead?

File: visualizer-server/config.toml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[server.grpc]
```
Comment (leviska) [inline visualizer-server/config.toml]: what's the idea difference between this and config/base.toml?

Comment (sevenzing) [reply visualizer-server/config.toml]: oops, forgot to add this file to gitignore

File: visualizer-server/build.rs (hunk 1 above)
Comment (sevenzing) [reply visualizer-server/build.rs]: ok, added `doctest = false` to `cargo.toml`

**Review by leviska — APPROVED** [review]

**Review by rimrakhimov — APPROVED** [review]

**Comment by rimrakhimov** [comment]
Can we make 'Doc tests' pass?

**Pull request merged**